examples/8a_benchmark_write_parallel.cpp

```diff
@@ -393,6 +398,15 @@ void parse(TestInput &input, std::string line)
         return;
     }
 
+    if (vec[0].compare("encoding") == 0)
+    {
+        if (vec[1].compare("f") == 0)
+            input.m_Encoding = openPMD::IterationEncoding::fileBased;
+        else if (vec[1].compare("g") == 0)
```

Comment:
A bit safer due to runtime checks:
```suggestion
    if (vec.at(0).compare("encoding") == 0)
    {
        if (vec.at(1).compare("f") == 0)
            input.m_Encoding = openPMD::IterationEncoding::fileBased;
        else if (vec.at(1).compare("g") == 0)
```